custom/icds_reports/tests/test_export_data.py

```diff
@@ -85,67 +85,106 @@ def test_children_export_headers(self):
             config={
                 'domain': 'icds-cas'
             },
+            beta=True
```

Comment:
I think that changing these tests like this means the currently released version is no longer tested

Reply:
So, In that case I should be having pairs of test case one to test with new change one to test the existing one right.

Reply:
yep

Reply:
and we can remove the duplicate tests when we release it from behind the feature flag